run.sh

```diff
@@ -11,26 +13,35 @@ if [ ! -n "$WERCKER_INSTALL_AWS_CLI_SECRET" ]; then
 fi
 
 if [ ! -n "$WERCKER_INSTALL_AWS_CLI_REGION" ]; then
-  echo '[WARN] No Region Specified'
+  echo '[WARN] No region specified'
 fi
 
+if [ ! -n "$WERCKER_INSTALL_AWS_CLI_VERSION" ]; then
+	echo '[WARN] No version specified, going with latest'
+  WERCKER_INSTALL_AWS_CLI_VERSION=""
```

Comment:
I realize now that our own indentation in this file isn't consistent, but if we're going to add lines with indentation, I'd rather they at least be consistent with themselves 😄 

If you're up to it, go ahead and update the indentation for this file to 2 spaces while you're at it.